docs/cv/.gitignore

```diff
@@ -1,3 +0,0 @@
-/systems.md
```

Comment:
These files/directories are only created virtually while mkdocs is running. No need to ignore them (anymore.)